src/modules/apps/e2e/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { render } from 'ink'
+
+import { apps, tester } from '../../../clients'
+import { RealTimeReport } from './report'
+import { getToken } from '../../../conf'
+import { ManifestEditor } from '../../../lib/manifest/ManifestEditor'
+import { ErrorBoundary } from './errorBoundary'
+import { TestRequest } from '../../../clients/Tester'
+
+
+export default async options => {
+  new EndToEndCommand(options).run()
```

Comment:
Since `.run()` returns a promise, this need to be:
```ts
export default async options => {
  await new EndToEndCommand(options).run()
}
```
Or

export default options => {
  return new EndToEndCommand(options).run()
}
```
The second is the more correct way.

Note that when an async function has just one promise it's not necessary for it to be `async`, just returning the promise will suffice - If you want to know more about promises in node just message me, we can do a call for me to show some cool examples that might help (helped me a lot when I was learning more about promises)